orchestra/communication/utils.py

```diff
@@ -0,0 +1,61 @@
+from orchestra.models import CommunicationPreference
+from orchestra.models import StaffBotRequest
+from orchestra.models import StaffingRequestInquiry
+from orchestra.models import StaffingResponse
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def mark_worker_as_winner(worker, task, required_role_counter,
+                          staffing_request_inquiry):
+    staffbot_request = StaffBotRequest.objects.filter(
+        task=task, required_role_counter=required_role_counter)
+
+    # Check whether staffbot request was sent out for this task
+    if not staffbot_request.exists():
+        return
+
+    staffbot_request = staffbot_request.first()
+    if staffing_request_inquiry:
```

Comment:
check this at the top, and put the code above in an `else` so we don't have to run queries that aren't necessary :)

Reply:
coo